src/components/uielements/assets/dynamicAsset/dynamicAsset.style.ts

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components'
+import { palette } from 'styled-theme'
+
+import { Size, Sizes, FontSizes } from './types'
+
+type ComponentProps = {
+  type?: string
+  size?: Size
+  className?: string
+}
+
+type ExtraProps = {
+  startCol: string
+  stopCol: string
+}
+
+type Props = ComponentProps & ExtraProps
```

Comment:
Why do you combine `Props` from two different types and not just define everything in `Props`?